DESCRIPTION

```diff
@@ -46,6 +46,8 @@ Suggests:
     rmarkdown,
     scales,
     shiny,
+    showtext,
```

Comment:
Since the packages may not be in validR, I have placed them in suggested packages and used the function `requireNamespace()` to assess if users have it installed before moving forward with the new fonts.